inst/specs/household.yml

```diff
@@ -0,0 +1,25 @@
+df:
```

Comment:
Este tibble é usado no R4DS (2e) para demonstrar o uso do .value na pivot_longer. Se usarmos data_nascimanto este (_) impacta o código no livro com resultado incorreto. Por isso, sugiro usarmos (.) ou outra forma sem adicionar um underline (_) extra no nome da coluna.

Codigo: 
` nucleo_familiar |> 
     pivot_longer(cols = !familia, 
     names_to = c(".value", "crianca"), 
     names_sep = "_", 
     values_drop_na = TRUE)`

Reply:
Bem observado!
Para ficar mais próximo ao livro, tem duas opções:
- `datanascimento_crianca1` sem separar o data nascimento
- `dn_crianca1` dn seria o equivalente ao dob (acho que esse fica mais parecido!!)
O que acha?

Reply:
dn_crianca fica mais proximo, porém diferente do "dob" em US, "dn" no BR não é muito utilizado. Sugiro ficar com "datanascimento_crianca1", pois apesar de não impactar neste exemplo específico, fica mais intuitivo caso a base for usado em outros lugares ou exemplos em treinamentos, etc. O que acha?

Reply:
Faz sentido, dob é uma sigla mais usada! Pode ser então. Vou alterar aqui! Muito obrigada!!